docker/test/upgrade/run.sh

```diff
@@ -77,6 +77,7 @@ remove_keeper_config "create_if_not_exists" "[01]"
 # it contains some new settings, but we can safely remove it
 rm /etc/clickhouse-server/config.d/merge_tree.xml
 rm /etc/clickhouse-server/config.d/enable_wait_for_shutdown_replicated_tables.xml
+rm /etc/clickhouse-server/config.d/zero_copy_destructive_operations.xml
```

Comment:
We need the same change for Tests Bugfix Validate Check:
https://s3.amazonaws.com/clickhouse-test-reports/57906/e4a831be17f85e345c5c9cd8f00bd22b8e759692/bugfix_validate_check.html
https://s3.amazonaws.com/clickhouse-test-reports/57906/e4a831be17f85e345c5c9cd8f00bd22b8e759692/stateless_tests_bugfix_validate_check.html
https://s3.amazonaws.com/clickhouse-test-reports/57906/e4a831be17f85e345c5c9cd8f00bd22b8e759692/stateless_tests_bugfix_validate_check/clickhouse-server.err.log

Reply:
https://github.com/ClickHouse/ClickHouse/pull/58078